starter-code/public/scripts/models/repo.js

```diff
@@ -11,8 +11,21 @@ var app = app || {};
     //       Remember that the callback function we'll want to call relies on repos.all
     //       being an array with a bunch of repo objects in it, so you'll need to
     //       populate it with the response from Github before you call the callback.
-
-  };
+    $.ajax({
+      url: `http://api.github.com/users/repos`,
+      method: 'GET',
+      headers: {
+        'Authorization': `token ${githubToken}`
+      }
+    }).then(function(data){
+        //=====> I don't think we need to map this data. sent it all to the array and filter it below after the call
+      repos.all = data.map((repo => {
+        name: repo.name
+      })
+      );
+      callback();
```

Comment:
This can also be put in a second .then (no need to fix this, just adding this comment as another way to do this!).